app/assets/javascripts/SourceCodeGlower/SourceCodeLineAnnotations.js

```diff
@@ -46,32 +46,39 @@ SourceCodeLineAnnotations.prototype.annotateLine = function(annotation_id, line_
 
   // Glow the Source Code Line
   var line = this.getLineManager().getLine(line_num);
-  line.glow();
-
-  // Add events so that when we mouse over this Source Code Line, we display
-  // the annotations
-  var me = this;
-
-  line.getLineNode().addEventListener('mouseover', function(event) {
-    me.displayTextsForLine(line_num, event.pageX, event.pageY);
-  });
-
-  line.getLineNode().addEventListener('mouseout', function(event) {
-    me.hideText();
-  });
+  var this_ref = this;
+  line.glow(annotation_id, column_start, column_end,
+    function(event) {
+      this_ref.displayTextsForLine(line_num, event, event.pageX, event.pageY);
```

Comment:
Identifier 'this_ref' is not in camel case.<br>Identifier 'line_num' is not in camel case.